pkg/ui/webpack.app.js

```diff
@@ -30,21 +31,12 @@ class RemoveBrokenDependenciesPlugin {
   }
 }
 
-let DashboardPlugin;
```

Comment:
@nathanstilwell , with current change, `make ui-watch` target will start two parallel webpack devservers (for cluster-ui and db-console) and I suggested that usage of Dashboard plugin isn't convenient here because it occupies entire screen and doesn't provide space for outputs from different processes.
Instead of it, I added `WebpackBar` plugin (the same as used in `cluster-ui`) to distinguish outputs.
What do you think about this change?